app/src/main/res/values-da/strings.xml

```diff
@@ -281,6 +281,9 @@
     The string is also used in the keyboard space bar to show the current keyboard language and in
     the keyboard languages panel list. -->
     <string name="settings_language_dutch">Hollandsk</string>
+    <string name="settings_language_galician">Galicisk</string>
+    <string name="settings_language_portuguese_br">Portugisisk (Brasilien)</string>
```

Comment:
@elima This line breaks the build because the value `"settings_language_portuguese_br"` appears twice in this document.